src/PHPUnit/Listeners/NaughtyTestListener.php

```diff
@@ -160,12 +169,15 @@ public function endTest(Test $test, $time)
     private function printNaughtyTest($name, array $modifications)
```

Comment:
since this doesn't print anymore, can you rename this to `addNaughtyTest()` or similar?